test/sql/deparse.sql

```diff
@@ -0,0 +1,57 @@
+
+
+\ir include/deparse_load.sql
+\o :TEST_OUTPUT_DIR/results/deparse_reference.out
+\d+ (public|"customSchema").*
+\o
+-- turn off alignment to avoid + signs at the end of lines
+\a
+-- tunr off headers
+\t
+SELECT format('DROP TABLE public.test1; %s', deparse_test) FROM _timescaledb_internal.deparse_test('public.test1');
+\gexec
+SELECT format('DROP TABLE public.test2; %s', deparse_test) FROM _timescaledb_internal.deparse_test('public.test2');
```

Comment:
Also we should toggle output switches so that the output of this command is not in the golden .out files. The reason is that correctness here is tested by the diff command at the end of the file. So we don't need to manually check the correctness of this output. By idea all content in .out files needs to be verified by all reviewers when PRs are submitted which is quite labor intensive and unecesssary  due to the diff.